translator/translate/otel/pipeline/host/translators.go

```diff
@@ -35,25 +41,36 @@ func NewTranslators(conf *confmap.Conf, os string) (pipeline.TranslatorMap, erro
 	})
 
 	switch v := conf.Get(common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.OtlpKey)).(type) {
-	case []interface{}:
+	case []any:
```

Comment:
just curious what is the difference here?

Reply:
Just a style difference.